bibliography.bib

```diff
@@ -546,6 +546,24 @@ @Article{clusterdata:Mishra2010
 
 ################ 2020
 
+@INPROCEEDINGS{clusterdata:Lin2020,
+   title = {Using GANs for Sharing Networked Time Series Data: Challenges, Initial Promise, and Open Questions},
+   author = {Lin, Zinan and Jain, Alankar and Wang, Chen and Fanti, Giulia and Sekar, Vyas},
+   year = {2020},
+   isbn = {9781450381383},
+   publisher = {Association for Computing Machinery},
+   address = {New York, NY, USA},
```

Comment:
Please use this for the address of the conference, not ACM.  Given that it was virtual, I'd just omit this. Thx.

Reply:
Done.